La'Tonia/stocks_challenge.py

```diff
@@ -0,0 +1,97 @@
+print("Challenge 3.2: Playing with the stock market")
+print()
+
+# Creating variables to store the current (approximate) market price of these 5 companies - Amazon, Apple, Facebook, Google and Microsoft.
+amazon = 3000
+apple = 100
+fb = 250
+google = 1400
+msft = 200
+
+print("Challenge 3.2.1: Taking user input")
+# TODO: Write code to ask the client his name and save it to a variable.
+client_name = input("Please enter your name: ")
+
+# TODO: Write code to ask the client his savings and save it to another variable.
+client_savings = input("Please enter the amount of your savings: ")
+
+# TODO: Write code to ask the client the stock he is interested in and save it to another variable, as shown below.
+# the above instruction is confusing to me given the code is/was already written. Commented it out. Rewrote it below. 
+# stock = input("Which stock are you interested in? Type 'amzn' for Amazon, 'aapl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft.")
+client_stock_interest = input("Which stock would you like to purchase? Type 'amzn' for Amazon, 'aapl' for Apple, 'fb' for Facebook, 'goog' for Google and 'msft' for Microsoft. ")
+print()
+
+print("Challenge 3.2.2: Perform user-specific calculations")
+# TODO: You have all 3 user inputs stored in variables. Based on that, write conditional (if-elif-else) statements to find out the number of stocks of the company that can be purchased with the savings amount.
+
+'''
+Your code should look like this:
+
+if stock == "amzn":
+    ...
+elif ...
+else ...
+'''
+# added code below after noting not sure how to separate just calculations from prints for customers... the code below is me trying to separate the two
+
+client_stock_calc = float(client_savings) / (amazon)
```

Comment:
Lets run this code tomorrow to test. Does line 37 set global "client_stock_calc" with amazon specifically?